### PR TITLE
Simplify shell to improve Menu transitions

### PR DESCRIPTION
**Pull request by takla21**
GitHub Issue: #

## PR Type

What kind of change does this PR introduce?
On the project I'm working on, we discovered that the way the menu was introduced causes a couple of animation issues. I'm bringing the fix from our project into the app template.

- Bugfix

## PR Checklist 
Please check if your PR fulfills the following requirements:

- [ ] ~~Interface members are XML documented~~
- [ ] ~~Documentation (XML or comments) has been added and/or existing documentation has been updated~~
- [ ] ~~[Architecture documents](./doc/Architecture.md) have been updated~~
- [ ] Tested on all relevant platforms

### CODE REVIEW

File: src/app/ApplicationTemplate.Shared.Views/Behaviors/MenuMarginBehavior.cs [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+namespace ApplicationTemplate.Views.Behaviors
+{
+	/// <summary>
+	/// This behaviors adjusts the margin of the control to set the height of the menu
+	/// when IsMarginEnabled == true. This way the page content is not hidden by the menu.
+	/// </summary>
+	public class MenuMarginBehavior
+	{
+		/// <summary>
```
Comment (takla21) [inline src/app/ApplicationTemplate.Shared.Views/Behaviors/MenuMarginBehavior.cs:11]: I need to either prove that tabbar is always 62 or make sure we get the actual height of the tabbar

File: src/app/ApplicationTemplate.Shared.Views/Content/Settings/SettingsPage.xaml [hunk 2]
```diff
@@ -14,7 +17,8 @@
 		<CommandBar Content="Profile"
 					x:Uid="Settings_CommandBar" />
 
-		<ScrollViewer Grid.Row="1">
+		<ScrollViewer Grid.Row="1"
```
Comment (takla21) [inline src/app/ApplicationTemplate.Shared.Views/Content/Settings/SettingsPage.xaml:20]: for some unknown reason to me at the moment I'm writing this, it's not working in this page (at least android)

Comment (Soap-141) [reply src/app/ApplicationTemplate.Shared.Views/Content/Settings/SettingsPage.xaml:20]: Any update on this? Did you try to update Uno?

Comment (takla21) [reply src/app/ApplicationTemplate.Shared.Views/Content/Settings/SettingsPage.xaml:20]: No updates really. At the time, I wanted to bring the whole logic with did on projects, but it was causing me too much problem so I've put it on the side. And I don't think I'll have time for this for a while...

Comment (takla21) [reply src/app/ApplicationTemplate.Shared.Views/Content/Settings/SettingsPage.xaml:20]: At this point, I could probably abandon this PR and probably find a better approach for the menu in the future

Comment (takla21) [reply src/app/ApplicationTemplate.Shared.Views/Content/Settings/SettingsPage.xaml:20]: I've closed it and deleted the branch

**Pull request closed without merging**